src/app/+collection-page/collection-page.component.html

```diff
@@ -32,6 +32,13 @@
                 </ds-comcol-page-content>
 
               </header>
+
+              <div class="add">
+                <a class="btn btn-lg btn-primary mt-1 ml-2" [routerLink]="['/submit']" [queryParams]="{collection:collection.id}" role="button">
```

Comment:
the `mt-1` and `ml-2` add the margin-top and margin-left that cause the button to be weirdly offset compared to the rest of the page. (copy pasted from a different place if I had to guess 😉). 

I'd remove them both and add a `mb-3` (for margin-bottom) instead to put some space between the button and the browse header below.

Reply:
Thanks, I didn't know what the various classes are, and copied this from the mydspace page
This was a temporary button until I moved it to the navigations, which I've adjusted now